oeps/oep-0004-arch-oauth-scopes.rst

```diff
@@ -31,7 +31,7 @@ The Open edX platform currently has the capability to act as an OAuth2 [#rfc6749
 
 Site admins can create long-lived OAuth2 client application keys that can in turn be exchanged for shorter-lived access and refresh tokens. These keys can optionally be scoped down to a custom list of available authorizations.
```

Comment:
With DOT, because we have https://courses.edx.org/api-admin/ this should be updated to indicate that anyone can create long-lived OAuth2 client applications.

Reply:
1. Note: Users can request access through the endpoint.
2. I'm not sure I can articulate why, but the OEP doesn't feel like the right place for this. 
If I were to add a sentence, it might look like: "Additionally, the LMS offers an /api-admin/ endpoint where any user can submit a request for a client application key."

Reply:
@feanil: FYI: Changes to this Scopes OEP are moving to a new PR: https://github.com/edx/open-edx-proposals/pull/150.  If you want to continue the conversation on the new PR, you can add a comment there and reference this thread.  Thanks.